app/controllers/groups_controller.rb

```diff
@@ -298,8 +298,7 @@ def members
   def add_members
     group = Group.find(params[:id])
     group.public_admission ? ensure_logged_in : guardian.ensure_can_edit!(group)
-
-    users = users_from_params
+    users = users_from_params(no_exception: true).to_a
```

Comment:
It's a little cumbersome to pass `no_exception` to a method like this. 

The "rails way" would be to add a bang `!` at the end if you want it to raise an exception or not. However in this case it looks like the method is only called in two actions, one which requires it and one which does not.

What I would recommend is never raising an exception in the method, but in the action that requires it check the return value, and if blank, raise the exception there.